src/EditorFeatures/Core/InlineRename/AbstractInlineRenameUndoManager.cs

```diff
@@ -97,6 +97,12 @@ public void Disconnect()
             this.RedoStack.Clear();
             this.initialState = null;
             this.currentState = null;
+
+            if (_trackedSession is not null)
```

Comment:
This fix is unrelated, just something I noticed while debugging. There were cases where we would disconnect and cause a nullref on session changing. This helps mitigate that. AFAIK we don't see many in the wild, so it could have been unique to pausing the debugger some how. Still safer code than before.